### PR TITLE
task 5 : functions

### PR DESCRIPTION
**Pull request by KateMzz**

### CODE REVIEW

**Comment by KateMzz** [comment]
how to get rid of ".idea"?
should I create ".gitignore" somewhere on my branch?

**Comment by MikitaTsiarentsyeu** [comment]
there is a gitignore for the repository, you could add .idea here

File: Tasks/Moroz/Task_5/Task_5.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        prev_n = n
+    b = sublst_checker(sublst, b)
+    return ", ".join(b)
+
+def sublst_checker(sublst: list, b: list):
+    if len(sublst) > 1:
+        b.append(f"{min(sublst)}-{max(sublst)}")
+    elif len(sublst) == 1:
+        b.append(f"{min(sublst)}")
+    return b
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Moroz/Task_5/Task_5.py:89]: very good, almost classical solution

**Review by MikitaTsiarentsyeu — APPROVED** [review]
good job, your score is 100

**Pull request merged**